website/src/components/Tooltip/keywordList.ts

```diff
@@ -0,0 +1,69 @@
+export interface GlossaryTerm {
+  link?: string;
+  content: string;
+}
+
+export const keywordList: Record<string, GlossaryTerm> = {
```

Comment:
I feel like it is a burden to maintain two files for the same term explaination. How about we create a  `website/scripts/generate-term.js`, and a command `'gen-term: node scripts/generate-term.js'`, we then run  `yarn gen-term` the script reads from the glossary.mdx and write each term a `term.json` file. the keywordList can read from this `term.json`? in that way, we only maintains the glossary.mdx file.